test/testcafe/framework/shared/index.js

```diff
@@ -29,3 +34,10 @@ export const Constants = {
   // https://devexpress.github.io/testcafe/documentation/guides/concepts/built-in-wait-mechanisms.html#wait-mechanism-for-xhr-and-fetch-requests
   TESTCAFE_DEFAULT_AJAX_WAIT: 3000, // 3seconds
 };
+export async function renderWidgetInLocale(locale) {
+  // re-render widget in a specific locale
+  await rerenderWidget({
+    'language':`${locale}`
+  });
+}
```

Comment:
Curious why you think we need this additional abstraction layer?

Reply:
My original intention was to have a shared `reRenderWidget` method without having to create a ClientFunction in each test file. Now that I am thinking about this. I am going to update this and call it `reRenderWidget` that takes a settings param.

```
export async function reRenderWidget(settings) {
  // re-render widget with settings
  await renderWidget(settings);
}

```